src/server/LockedQueue.hpp

```diff
@@ -6,7 +13,7 @@
 /**
  * Simple mutex-guarded queue
  */
-namespace Network
+namespace network
 {
     template <typename _T> class LockedQueue {
```

Comment:
`_T` is a reserved template name
please use T instead

```suggestion
     template <typename T> class LockedQueue {
```